v3/src/v2/codap-v2-document.ts

```diff
@@ -161,17 +177,30 @@ export class CodapV2Document {
   }
 
   registerCases(data: IDataSet, cases: ICodapV2Case[], level: number) {
+    const v2CollectionInfo = this.v2CaseIdInfoArray[level]
+    const groupKeyCaseIds = v2CollectionInfo.groupKeyCaseIds
     cases.forEach(_case => {
-      const { guid, values } = _case
+      // some v2 documents don't store item ids, so we generate them if necessary
+      const { guid, itemID = v3Id(kItemIdPrefix), values } = _case
```

Comment:
I found that the v2 itemIDs were typed as numbers, but in the document I was testing they were strings. I didn't check this branch out locally to test it, but I just wanted to point that out incase it was going to cause problems.

Reply:
I think typing them as`number` was just an error because as far as I know they've always been strings.